File: Data Structures & Algorithms/duplicate-integer/submission-1.go
func hasDuplicate(nums []int) bool {
	mynumsMap := make(map[int]bool)
	for _, e := range nums {
		if mynumsMap[e] {
			return true
		} else {
			mynumsMap[e] = true
		}
	}
	return false
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        counts = {}
        for n in nums:
            counts[n] = counts.get(n,0)+1
        
        i = 0
        for j in sorted(counts):
            for _ in range(counts[j]):
                nums[i] = j
                i += 1

        top_k = heapq.nlargest(k, counts.items(), key=lambda x: x[1])
        result = [item[0] for item in top_k]
        return result
                